superset-frontend/src/components/Chart/Chart.tsx

```diff
@@ -214,7 +209,7 @@ class Chart extends React.PureComponent<ChartProps, {}> {
     });
   }
 
-  renderErrorMessage(queryResponse: queryResponse) {
+  renderErrorMessage(QueryResponse: QueryResponse) {
```

Comment:
The argument should be lowercase, only the type is PascalCase

Reply:
renderErrorMessage(queryResponse: QueryResponse)